src/librustc_llvm/ffi.rs

```diff
@@ -49,6 +49,10 @@ pub enum CallConv {
 
 /// LLVMLinkage
 ///
+/// This does not attempt to mirror the similar enum in LLVM, as this is fragile
```

Comment:
nit: comment should be just `/// LLVMRustLinkage`  - see e.g. `LLVMRustAsmDialect`. While you're at it, maybe also convert the other non-LLVMRust enums before they bite us?